src/test/kotlin/net/atos/zac/policy/output/PolicyOutputFixtures.kt

```diff
@@ -5,67 +5,101 @@
+) = DocumentRechten(
+    lezen, wijzigen, verwijderen, vergrendelen, ontgrendelen, ondertekenen,
+    toevoegenNieuweVersie, verplaatsen, ontkoppelen, downloaden
 )
 
-fun createZaakRechten() = ZaakRechten(
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
-    true,
+fun createTaakRechtenAllDeny(
+    lezen: Boolean = false,
+    wijzigen: Boolean = false,
+    toekennen: Boolean = false,
+    creeerenDocument: Boolean = false,
+    toevoegenDocument: Boolean = false,
+) = TaakRechten(lezen, wijzigen, toekennen, creeerenDocument, toevoegenDocument)
+
+fun createZaakRechtenAllAllow() = ZaakRechten(
+    true, true, true, true, true, true, true,
+    true, true, true, true, true, true,
+    true, true, true, true, true,
+    true, true, true, true,
+    true, true, true, true, true
+)
+
+@Suppress("LongParameterList")
+fun createZaakRechtenAllDeny(
+    lezen: Boolean = false,
+    wijzigen: Boolean = false,
+    toekennen: Boolean = false,
+    behandelen: Boolean = false,
+    afbreken: Boolean = false,
+    heropenen: Boolean = false,
+    wijzigenZaakdata: Boolean = false,
+    wijzigenDoorlooptijd: Boolean = false,
+    verlengen: Boolean = false,
+    opschorten: Boolean = false,
+    hervatten: Boolean = false,
+    creeerenDocument: Boolean = false,
+    toevoegenDocument: Boolean = false,
+    koppelen: Boolean = false,
+    koppelenGerelateerd: Boolean = false,
+    versturenEmail: Boolean = false,
+    versturenOntvangstbevestiging: Boolean = false,
+    toevoegenInitiatorPersoon: Boolean = false,
+    toevoegenInitiatorBedrijf: Boolean = false,
+    verwijderenInitiator: Boolean = false,
+    toevoegenBetrokkenePersoon: Boolean = false,
+    toevoegenBetrokkeneBedrijf: Boolean = false,
+    verwijderenBetrokkene: Boolean = false,
+    toevoegenBagObject: Boolean = false,
+    startenTaak: Boolean = false,
+    vastleggenBesluit: Boolean = false,
+    verlengenDoorlooptijd: Boolean = false,
+) = ZaakRechten(
+    lezen, wijzigen, toekennen, behandelen, afbreken, heropenen, wijzigenZaakdata, wijzigenDoorlooptijd,
+    verlengen, opschorten, hervatten, creeerenDocument, toevoegenDocument, koppelen, koppelenGerelateerd,
+    versturenEmail, versturenOntvangstbevestiging, toevoegenInitiatorPersoon, toevoegenInitiatorBedrijf,
+    verwijderenInitiator, toevoegenBetrokkenePersoon, toevoegenBetrokkeneBedrijf, verwijderenBetrokkene,
+    toevoegenBagObject, startenTaak, vastleggenBesluit, verlengenDoorlooptijd
 )
 
-fun createWerklijstRechten(
-    inbox: Boolean = true,
-    ontkoppeldeDocumentenVerwijderen: Boolean = true,
-    inboxProductaanvragenVerwijderen: Boolean = true,
-    zakenTaken: Boolean = true,
-    zakenTakenVerdelen: Boolean = true
+fun createWerklijstRechtenAllDeny(
```

Comment:
I prefer leaving the original function: `createWerklijstRechten` (with all policies defaulting to true).  Creating an additional function `createWerklijstRechtenAllDeny` is ok I guess but that should then just call the `createWerklijstRechten` function I would say. I think that is much cleaner. I prefer to have `createXXX` methods for all model classes that we use in tests with sensible defaults which you can then override if you need to in a particular test.

Reply:
Or you could just have one method `createWerklijstRechten` which takes an additional argument: `defaultAllToFalse` which defaults to false but if set to true sets all those policies to false instead of true. That's even nicer I think.

Reply:
Same for the other createXXXRechten functions.

Reply:
I modified tests to use only specific policies. Which makes setting all policies to true redundant.

I can get back the old names, but do we want to keep all-open/true behaviour as well?

Reply:
I don't know. I think sensible defaults are the most logical for these fixture methods and I would say that all-true is the most sensible default so yes, I would personally do that, even if we don't use all defaults right now?

Reply:
Added AllDeny fixtures